packages/block-editor/src/components/off-canvas-editor/style.scss

```diff
@@ -14,6 +14,10 @@
 	}
 }
 
+.offcanvas-editor-appender__description {
+	display: none;
```

Comment:
Question: does not `display: none` make the content in accessible to screen readers _entirely_? Or is there some exception when we have the describedBy -> ID relationship explicitly stated in the ARIA?

Reply:
I think `display: none` is fine for aria descriptions. For visually hidden text where a different approach needs to be taken.

Reply:
Thanks for confirming.